docs/known-issues.md

```diff
@@ -14,6 +14,7 @@ This document lists known issues and bugs in GCSFuse, their impact, and the rele
 
 ## Resolved Issues
 
-| Issue | Affected Versions | Fixed in | Reference |
-| :--- | :--- | :--- | :--- |
-| Input/output error when metrics are enabled. | Applications may receive input/output errors from GCSFuse mounts when metrics are enabled. | v2.12.0 | [#3870](https://github.com/GoogleCloudPlatform/gcsfuse/issues/3870) |
+| Issue                                                                                                      | Affected Versions                                                                                                                                                                                                                                                                                                               | Fixed in | Reference |
+|:-----------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| :--- | :--- |
+| Input/output error when metrics are enabled.                                                               | Applications may receive input/output errors from GCSFuse mounts when metrics are enabled.                                                                                                                                                                                                                                      | v2.12.0 | [#3870](https://github.com/GoogleCloudPlatform/gcsfuse/issues/3870) |
+| Writing to a file fails with an Input/Output error on the application side, accompanied by 503 errors in the GCSFuse logs. This occurs when streaming writes are enabled. | This issue was caused by stalls during write operations. It has been resolved in GCSFuse v3.3.0. Users should upgrade to v3.3.0 or later. **Workaround:** Disable streaming writes (`--enable-streaming-writes=false`) only if user can't upgrade. This flag reliably prevents the error only when used on VMs with Local SSDs. |v3.3.0||
```

Comment:
`This flag reliably prevents the error only when used on VMs with Local SSDs.` - Not very clean, can we make more actionable. I mean, `--temp-dir` has SSD path.